assignment/scripts/describe.js

```diff
@@ -123,47 +119,45 @@ if (number >= 2) {
 
 */
 
-
-
-//4. We start with three variables -- colorOne is set to 'blue' and 
-// colorTwo is set to 'red', and mix is set to true. We check if mix is true 
+//4. We start with three variables -- colorOne is set to 'blue' and
+// colorTwo is set to 'red', and mix is set to true. We check if mix is true
 // -- it is, so we set colorOne and colorTwo to 'purple'
 
 /*
-let colorOne = 'red';
-let colorTwo = 'blue';
+let colorOne = 'blue';
+let colorTwo = 'red';
```

Comment:
The assignment was to add comments describing hypothetical fixes, but this approach makes sense to me.